llama-index-integrations/indices/llama-index-indices-managed-postgresml/pyproject.toml

```diff
@@ -0,0 +1,66 @@
+[build-system]
+build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core"]
+
+[tool.codespell]
+check-filenames = true
+check-hidden = true
+skip = "*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt,*.ipynb"
+
+[tool.llamahub]
+contains_example = false
+import_path = "llama_index.indices.managed.postgresml"
+
+[tool.llamahub.class_authors]
+PostgresmlIndex = "llama-index"
+
+[tool.llamahub.secondary_class_authors]
```

Comment:
hmm, I think all the classes + authors can be listed under `[tool.llamahub.class_authors]` (feel free to put your actual GitHub username if you want!)

Reply:
Sorry I'm not very familiar with pyproject.tomls, what should this look like?

Reply:
```
[tool.llamahub.class_authors]
class_name_1 = "github username"
class_name_2 = "github ussrname" 
```

And so on